docs/CONTRIBUTING.md

```diff
@@ -1,16 +1,7 @@
-# Contribution Guidelines
-
-## Contents
-
-- [Code of Conduct](#code-of-conduct)
```

Comment:
Hm, having a ToC in markdown files is useful when browsing via GitHub.

Reply:
But the new docs rendered creates this by default. Unfortunately we cannot have both.